cmd/jaeger/internal/integration/e2e_integration.go

```diff
@@ -46,7 +47,7 @@ type E2EStorageIntegration struct {
 // This function should be called before any of the tests start.
 func (s *E2EStorageIntegration) e2eInitialize(t *testing.T, storage string) {
 	logger := zaptest.NewLogger(t, zaptest.WrapOptions(zap.AddCaller()))
-	configFile := createStorageCleanerConfig(t, s.ConfigFile, storage)
+	configFile := createStorageCleanerConfig(t, s.SkipStorageCleaner,s.ConfigFile, storage)
```

Comment:
Why pass the flag to the function? Just check it here and don't introduce unnecessary changes to the code.